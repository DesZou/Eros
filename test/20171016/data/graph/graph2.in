5 6 4
4 2 825
3 1 23
4 5 570
3 2 279
3 4 502
5 1 216
10
5 1
3 5
5 3
1 2
1 4
3 4
5 4
4 2
1 5
5 1
