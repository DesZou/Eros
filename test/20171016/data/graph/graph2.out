1697
285
285
348
571
1150
1343
827
1697
1697
